comp/otelcol/otlp/components/exporter/serializerexporter/exporter.go

```diff
@@ -143,12 +131,7 @@ func NewExporter(
 	hostGetter SourceProviderFunc,
 	statsIn chan []byte,
 ) (*Exporter, error) {
-	// Log any warnings from unmarshaling.
-	for _, warning := range cfg.warnings {
-		set.Logger.Warn(warning)
-	}
-
-	tr, err := translatorFromConfig(set, attributesTranslator, cfg, hostGetter, statsIn)
+	tr, err := translatorFromConfig(set, attributesTranslator, cfg.Metrics.Metrics, hostGetter, statsIn)
```

Comment:
are there no warnings possible  with the new package or is it just not worth it to log them?

Reply:
This is handled in `checkAndCastConfig` in `comp/otelcol/otlp/components/exporter/datadogexporter/factory.go`